lib/stones-spec/language/gobstones.rb

```diff
@@ -28,6 +28,10 @@ def self.is_runtime_error?(result)
       result.include? 'Error en tiempo de ejecución'
     end
 
+    def self.is_syntax_error?(result)
+      result.include? 'Error de sintaxis' or result.include? 'Error en el programa'
```

Comment:
Ok, I used the "is" prefix because there was an "is_runtime_error?" method. I changed both.